### PR TITLE
[WIP][ZEPPELIN-2726] Move angular related pkgs from bower to npm (zeppelin-web)

### PR DESCRIPTION
**Pull request by 1ambda**
### What is this PR for?

Move angular related packages from bower to npm. Because bower team recommends using npm (or yarn) instead of bower. It will not be maintained.

For angular-websocket, can't move it to npm currently. As it's tightly tangled with node packages like (ws, fs, tls, ...) which are not available on the browser side. We can solve this problem after applying webpack2+

- https://github.com/AngularClass/angular-websocket/issues/82

### What type of PR is it?
[Improvement]

### Todos
* [x] - Move angular related packages 
* [x] - DO NOT use angular as the global dependency
* [x] - remove not used global names from .eslintrc

### What is the Jira issue?

[ZEPPELIN-2726](https://issues.apache.org/jira/browse/ZEPPELIN-2726)

### How should this be tested?

CI will test it, all existing functionalities should work properly.

### Screenshots (if appropriate)

NONE

### Questions:
* Does the licenses files need update? - NO
* Is there breaking changes for older versions? - NO
* Does this needs documentation? - NO

### CODE REVIEW

**Comment by 1ambda** [comment]
@soralee Could you help review thid?

**Comment by soralee** [comment]
It looks sounds good! 
Sure, Let me test and check this branch.

File: zeppelin-web/src/index.html [hunk 1]
```diff
@@ -124,16 +121,9 @@
...
     <script src="bower_components/json3/lib/json3.js"></script>
     <script src="bower_components/bootstrap/dist/js/bootstrap.js"></script>
-    <script src="bower_components/angular-cookies/angular-cookies.js"></script>
-    <script src="bower_components/angular-sanitize/angular-sanitize.js"></script>
-    <script src="bower_components/angular-animate/angular-animate.js"></script>
-    <script src="bower_components/angular-touch/angular-touch.js"></script>
-    <script src="bower_components/angular-route/angular-route.js"></script>
-    <script src="bower_components/angular-resource/angular-resource.js"></script>
-    <script src="bower_components/angular-bootstrap/ui-bootstrap-tpls.js"></script>
+    <script src="bower_components/angular/angular.js"></script>
```
Comment (sravan-s) [inline zeppelin-web/src/index.html]: Do we still need angular like this?

Comment (1ambda) [inline zeppelin-web/src/index.html]: because angular-websocket requires it.

**Comment by 1ambda** [comment]
Thanks for the review. @soralee @sravan-s 

we are using annotation for angular, but two angular conflicts. (bower side required by angular-websocket) 

Thus, I will change this PR as WIP and comment after the fix. 

- applying webpack2 both for zeppelin-web and helium
- remove angular-websocket from bower using webpack2

**Comment by 1ambda** [comment]
Will be handled later

**Pull request closed without merging**